src/Components/Admin/AssetModelViewer/objectTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ObjectTypes {
```

Comment:
It looks like you added the enum here properly, let's use it in the code! When we do `setObjectType`, you'll need to add a switch statement to check for "object", "background" and then a default which can throw an error.